release/src/main/scripts/run_rc_validation.sh

```diff
@@ -547,4 +568,49 @@ if [[ $confirmation = "y" ]]; then
   fi
 fi
 
+echo "[Current Task] Run All PostCommit Tests against Release Branch"
+echo "This task will create a PR against apache/beam."
+echo "After PR created, you need to comment following phrase in the created PR:"
+echo "1. comment 'Run Go PostCommit'."
```

Comment:
I worry that the set of post-commits and trigger phrases will evolve over time and this list will get stale.

That being said, I don't have a good solution for keeping these up-to-date. I wish there were a way to pull the trigger phrases from Jenkins or the jenkins job configs in our build. Or maybe we could make our job DSL code for grep-able for trigger phrases.

At any rate, for this PR, maybe add a comment / echo to say something like "Verify the list of trigger phrases in postcommit jobs inside `.test-infra/jenkins`

Reply:
I feel like, it's not possible to make everything up-to-date automatically. I would prefer adding instruction into release guide to say, if the phrase is out-of-date, you can find the latest one in .test-infra/jenkins and please update the script.

Reply:
I am new to this, haven't used the release automation, but what about this: `.test-infra/jenkins/job_ReleaseVerification` that runs what you want. Then you can go to the Jenkins UI and run it against the branch or a specific commit. And then this script would be fine to phrase trigger that verification.

I guess this is just a layer of indirection using Jenkins to aggregate the dependencies, and we can use gradle or bash too. Hrm

Reply:
FWIW 600 lines of bash is approaching the maximum size before I'd want to use a different tech...

Reply:
> I am new to this, haven't used the release automation, but what about this: `.test-infra/jenkins/job_ReleaseVerification` that runs what you want. Then you can go to the Jenkins UI and run it against the branch or a specific commit. And then this script would be fine to phrase trigger that verification.
> 
> I guess this is just a layer of indirection using Jenkins to aggregate the dependencies, and we can use gradle or bash too. Hrm

My concern here is, if we use one job to run all postcommit and precommit for validation purpose, then there will beall gradle build logs inside one jenkin build. It may be difficult for a dev to find which postcommit or percommit fails.

Reply:
> FWIW 600 lines of bash is approaching the maximum size before I'd want to use a different tech...

Yes this is a issue. The reason why we chose bash script from the first place is, we want to make every step could be ran manually. In this case, the bash script would fit into this requirement. Besides, there are many manual steps for validation, kind of hard to packet into other tech stack.